src/css/base/common.css

```diff
@@ -22,7 +22,7 @@ section {
   padding-bottom: 80px;
 }
 
-.hero::before {
+section::before {
```

Comment:
Typo, should be main :)

Reply:
this solution should be better: scroll-padding-top: var(--header-height);